### DISCUSSION TITLE
Slaughterfest feedback

### DISCUSSION THREAD

**Discussion by Leroy8888** [HelixNebulaStudio/RiseOfTheDead#721] [Ideas]
A few tests were run in dev branch to see how the current SF functions. During it we ran across a few topics seem fit for a discussion page / thread.

It seems that if players tie with candy (somehow) the first player will be incorrectly displayed in the feed. Unsure to how it would appear to other players which aren't in first place, but if the player is then they will always appear to be first on their own device.

The percentage yield you get from killing players carrying candy seems to be fairly low and doesn't seem to reach 40% (the max without clothing) as quickly. For example killing a player with 400 candies will drop 25% of said amount, killing one with 600 (or more) will drop 40%. 

The mission `Slaughter King` seems a bit lackluster in the rewards. It currently gives 25 perks and a few tweaks, but can take up to 10 minutes if you already join a match that is in progress, since you would start at a disadvantage. Compare this to the mission escort which takes roughly 30 seconds and gives the same reward.

SF itself also doesn't really seem to have a incentive at the moment to be there. Besides one mission there isn't really anything to gain there besides some fun with pvp. But the small playerbase makes this difficult and it's likely most player would opt to just skip `slaughterfest king`. 

I think some sort of rewards for SF as it currently is would draw more players to it. As for the mission tied it, it should be increased. Maybe also make it a bit easier, like for example allowing the first 3 places to complete the mission and increasing the perk reward to either 75 or 100, maybe even give it a special tier for events? Meanwhile, SF itself could also give perks or tweaks as reward when reaching those places. It's something not related to the event, but it could draw a few more players to the event, especially newer player that would like perks or older player for tweaks. These _could_ also be rewarded in the form of tomes or some other consumable that grants perks.

Also as question, is SF now permanently available? If so, then maybe the rewards mentioned above should be kept exclusive to the season. It should give a reason to play it, but it should not be a way in regular gameplay to quickly accumulate perks without progressing though either the story or grind. I also think the mission `Slaughter king` should be exclusive to halloween as it would be very unlikely to always have people playing in there especially after halloween.

**Comment by MrGarrettFerret** [comment]
I do agree that Slaughter King's rewards are quite bad for the amount of time it takes, and I agree that it should be buffed. Maybe make it the first tier 4 mission or something?

Having consumables drop from getting first/second/third place in the Slaughterfest would be a really nice way to incentivize people to play it. I would really like for it to become a permanent mode though. PvP in RotD is actually really fun and I think it should stay year-round (You could also use it to test some parts of Almonds PvP).

I disagree with having the rewards from the mode be limited to only the season though. Slaughterfest keeps people playing during the Halloween event, and this is because it has rewards that you can keep earning again and again by playing. If these rewards were to be removed, no one would play the mode because there'd be no incentive. The mode should have rewards year-round to keep people playing RotD. The game has always struggled with replayability, once you get maxed out weapons and level 560 there's not really a reason to keep playing. However, Slaughterfest can help fix this issue by giving out things like consumables or something else that keeps players engaged with the game. The mode could be incorporated into the grind, allowing dedicated players to get to the endgame and access new update content much quicker and more enjoyably than shooting the same zombies outside of Sunday's for 6 hours straight.

Slaughter King should also be kept year-round. It would give people another reason to play the gamemode, which kinda defeats the argument of there not being anyone in there to play after the event.

**Comment by MXKhronos** [comment]
@Leroy8888 @MrGarrettFerret 

Thanks for all the feedback. I'll keep this short as I'll write another dedicated dev log for the current developing update.

- Slaughterfest is intended to be a casual fun gamemode only for halloween, I'm working on a new gamemode which I'll talk more about in another post.
- @Leroy8888 Candy drop mechanics has nuanced to it. Each round is 8 minutes long, and for every 2 seconds, a candy is secured. Meaning it can no longer be dropped. So if you have 500 candies by 4 minutes, (4x60/2)=120 candies are secured and the other (500-120)=380 candies can be dropped, so if the killer has a halloween mask, the victim will drop (380x0.5)=190 candies on death at the moment. By the last minute of the round, every player cannot drop below 210 candies when killed. The parameters are open to adjustment, but the mechanics is to weaken candy drop later on in the round while remaining valuable to target the top 3.
- `Slaughter King` is a halloween exclusive board mission and will remain so.
- Noted on `Slaughter King`'s reward, and I'll add an additional reward item for the mission.
- `Slaughter King` will now just require you to be top 3 to complete.
- Fixing some issues where the ui updates the end game top 3 placements when players leave.

If I missed anything let me know. This halloween event I hope to prioritize replayability via gameplay rather than replayability for rewards,
